scripts/create-sentry-release

```diff
@@ -22,12 +22,11 @@ sentry-cli --version
 
 echo 'Pulling debug information...'
 gsutil cp \
-  "gs://dicd-team-devinfra-cd--symbolicator/difs/${VERSION}/symbolicator-debug.zip" \
-  "gs://dicd-team-devinfra-cd--symbolicator/difs/${VERSION}/symbolicator.src.zip" \
+  "gs://dicd-team-devinfra-cd--symbolicator/difs/${VERSION}/symbolicator-*.zip" \
```

Comment:
This isn't quite right, the second zip above doesn't have a `-`. It should be
```suggestion
  "gs://dicd-team-devinfra-cd--symbolicator/difs/${VERSION}/symbolicator*.zip" \
```

Reply:
Not sure I follow should be `symbolicator-${{ matrix.arch }}` and `symbolicator-${{ matrix.arch }}-debug` but indeed just the * without the - should also still match

Reply:
Oh shoot, sorry, I missed that you added hyphens above. My bad.

Reply:
No worries